microsoft/testsuites/core/floppy.py

```diff
@@ -20,27 +18,23 @@ class Floppy(TestSuite):
     @TestCaseMetadata(
         description="""
         This test case will
-        1. Ensure the floppy driver is not loaded directly
-        into the kernel by checking boot config
-        2. Ensure the floppy driver is not loaded as a module using lsmod
+        1. Dry-run modprobe to see if floppy module can be loaded
+        2. If "insmod" is executed then it is not already loaded
+        3. If module cannot be found then it is not loaded
```

Comment:
1. The test need to explain the test point, for example, "verify the floppy driver should be disabled in Azure"
2. Explain the reason from a link.
3. Clarify what's expected, what's not. for example, if it's "not loaded". Does it mean the distro is in good status?